index.html

```diff
@@ -2142,6 +2142,14 @@ <h3>Securing Data Losslessly</h3>
 `DATA_LOSS_DETECTION_ERROR`, when data is dropped by a JSON-LD processor,
 such as when an undefined term is detected in an input document.
           </p>
+
+          <p>
+Similarly, since <a>conforming secured documents</a> can be transferred from one
+security domain to another, <a>conforming processors</a> that process the
+<a>conforming secured document</a> cannot assume any particular base URL for
+the document. When serializing to RDF, implementations MUST ensure that the
+base URL is set to null.
```

Comment:
```suggestion
When serializing to RDF, implementations MUST ensure that the
base URL is set to null.
```

This can be a bit misread as if the base URL was a particular term in RDF that has to be set to null. This isn't the case, RDF does not have this notion (only serializations like JSON-LD or Turtle have).  I wonder whether this sentence serves any useful purpose, actually: the previous sentence does say that "cannot assume any particular base URL for the document", which sounds enough to me.

Reply:
A normative MUST ensures that we can test this functionality in the test suite. We want to test to make sure that relative URLs aren't used and/or are detected to increase interoperability. If we don't have the normative MUST statement, we won't have a test for this, and interop will suffer, IMHO.